ci/run_clang_tidy.sh

```diff
@@ -40,8 +40,15 @@ function exclude_chromium_url() {
   grep -v source/common/chromium_url/
 }
 
+# It gets a bit weird having the compilation database include the clang_tools
```

Comment:
Perhaps we should exclude this in gen_compile_db script, with option, so you get clang-tidy check on tools.